lib/manageiq/appliance_console/cli.rb

```diff
@@ -141,6 +145,7 @@ def parse(args)
         opt :iparealm,      "IPA Server realm (optional)", :type => :string
         opt :ca,                   "CA name used for certmonger",       :type => :string,  :default => "ipa"
         opt :timezone,             "Time zone",                         :type => :string
+        opt :datetime,             "Date and time, in YYYY-MM-DD_HH:MM:SS format", :type => :string
```

Comment:
Couldn't we use a more widespread format here? Maybe [ISO8601](https://en.wikipedia.org/wiki/ISO_8601)?